modules/cert-manager-configure-cloud-credentials-aws-sts.adoc

```diff
@@ -0,0 +1,166 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-aws.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-aws-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} for the AWS Security Token Service cluster
+
+You can configure cloud credentials for the {cert-manager-operator} on the AWS Security Token Service (STS) cluster by using the `ccoctl` binary.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. Create a directory to store the `CredentialsRequest` custom resource (CR) of the {cert-manager-operator} by running the following command:
```

Comment:
```suggestion
. Create a directory to store the `CredentialsRequest` custom resource (CR) by running the following command:
```
As CredentialsRequest is not a cert-manager operator specific resource, we should not include the mention of it with the CR.